src/services/UserService.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+import humps from 'humps';
+import api from '../config/api';
+import type { User } from '../models/User';
+
+export function authHeader() {
+  // return authorization header with jwt token
+  const user = JSON.parse(localStorage.getItem('user'));
+
+  if (user && user.token) {
+    return { 'X-CSRF-TOKEN': user.token };
+  }
+  return {};
+}
+
+class UserService {
+  static async get(id: string): Promise<User> {
+    const headers = authHeader();
+    const response = await api.get(`/users/${id}`, {}, { headers });
+    return response.data;
+  }
+
+  static async list(): Promise<User[]> {
+    const headers = authHeader();
+    const response = await api.get('/users', {}, { headers });
+
+    return response.data;
+  }
+
+  static async create(formData: FormData): Promise<User> {
+    try {
+      const response = await api.post('/users', formData);
+      return response.data;
+    } catch (e) {
+      throw e;
+    }
+  }
+
+  static async login(emailOrUsername: string, password: string): Promise<User> {
+    try {
+      const response = await api.post('/users/login', humps.decamelizeKeys({ emailOrUsername, password }), { withCredentials: true });
+      localStorage.setItem('user', JSON.stringify(response.data));
```

Comment:
Ah, I see the user is not saved in the store. But why not?

Reply:
To access it in the user service, see above